bitnami/mongodb-sharded/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb-sharded
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb-sharded
   - https://mongodb.org
-version: 3.6.0
+version: 3.6.1
```

```suggestion
version: 3.7.0
```

Increase a minor since we are adding new functionality.

---

Done - wasn't sure if this was a bug fix (since it was already documented) or a new feature.